app/app.go

```diff
@@ -135,8 +137,20 @@ func MakeCodec() *codec.Codec {
 	return cdc.Seal()
 }
 
-// https://github.com/cosmos/sdk-tutorials/blob/c6754a1e313eb1ed973c5c91dcc606f2fd288811/app.go#L73
+func ChainAnteChains(chains ...sdk.AnteHandler) sdk.AnteHandler {
```

Comment:
this can be private.  It really joins antes, maybe:

```go
func mergeAnteHandlers(handlers ...sdk.AnteHandler) sdk.AnteHandler {
```

Reply:
Either option will go. Reason why I did it this way is compatibility with sdk API. If they allow at some point to append handlers to default chain we won't need to change fn signatures if go with 2nd option

```go
app.SetAnteHandler(
  mergeAnteHandlers(
    auth.NewAnteHandler(),
    dante.NewHandler(),
  ),
)
```

```go
app.SetAnteHandler(
  mergeAnteHandlers(
    auth.NewAnteHandler(),
    sdk.ChainAnteDecorators(
        dante.NewHandler(),
    ),
  ),
)
```

Reply:
right on, thanks.  your call.